README.md

```diff
@@ -1,28 +1,40 @@
 # Blender For Unreal Engine
-This Add-on allows you to export content created with Blender to Unreal Engine. StaticMeshs, SkeletalMeshs, Animations (NLA and Actions), Collisions and Sockets, Alembic animations, Camera and sequencer [...]
+
+This Add-on allows you to export content created with Blender to Unreal Engine. StaticMeshs, SkeletalMeshs, Animations (NLA and Actions), Collisions and Sockets, Alembic animations, Camera and sequencer...
+

```

Comment:
Fixed the issue due to the IDE issuing warnings such as ‘do not write raw URLs’.